05-visualisation-ggplot2.Rmd

```diff
@@ -185,6 +185,7 @@ sections and points are counted wihin hexagons. The number of points per hexagon
 is encoded by color.
 
 ```{r}
+library(hexbin)
```

Comment:
If we need to load extra libraries for this call to work, that should go into into its own code chunk somewhere near the beginning. Unless you unload the library right after the plot invocation, but that would seem silly here.